content/data-reporting/reports-api/v3.4/index.mdx

```diff
@@ -84,8 +159,10 @@ Shows the average chatting duration of agents within a license.
 <Sample>
 
 ```shell
-curl "https://api.livechatinc.com/v3.4/reports/chats/agents_chatting_duration?from=2020-09-01T00:00:00%2B02:00&to=2020-09-14T23:59:59%2B02:00" \
-  -H 'Authorization: Bearer <your_access_token>'
+curl "https://api.livechatinc.com/v3.4/reports/chats/agents_chatting_duration" \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{}'
```

Comment:
Don't we want to put any filters in the example?

Reply:
As Ola said filters example is presented below params table and i do not see huge value in duplicating this in each report

Reply:
It just seems odd to me to describe the filters and then show the one use case which doesn't use them at all

Reply:
Added sample request payloads